### PR TITLE
Replace sa icon to taskbar

### PR DESCRIPTION
**Pull request by G-Moris**
Скриншот 26-12-2024 020647
Скриншот 26-12-2024 020722
Скриншот 26-12-2024 020753

### CODE REVIEW

**Comment by ghost** [comment]
It's the smallest details that make the difference, and that's great of you

**Comment by Fernando-A-Rocha** [comment]
Thank you. This is awesome. Gives a unique touch to MTA that was needed.

File: Client/core/DXHook/CProxyDirect3D9.cpp [hunk 1]
```diff
@@ -172,6 +175,23 @@ HRESULT CProxyDirect3D9::CreateDevice(UINT Adapter, D3DDEVTYPE DeviceType, HWND
...
     DwmSetWindowAttribute(hFocusWindow, DWMWA_USE_IMMERSIVE_DARK_MODE, &darkTitleBar, sizeof(darkTitleBar));
 
+    // Update icon
+    if (HICON icon = LoadIcon(g_hModule, MAKEINTRESOURCE(IDI_ICON1)))
+    {
+        const auto paramIcon = reinterpret_cast<LPARAM>(icon);
+        for (const WPARAM size : {ICON_SMALL, ICON_BIG})
+        {
+            SendMessage(hFocusWindow, WM_SETICON, size, paramIcon);
+        }
```
Comment (botder) [inline Client/core/DXHook/CProxyDirect3D9.cpp:185]: I don't really mind this, but it's overkill for you could've just duplicated the SendMessage line

**Review by botder — APPROVED** [review]

File: Client/core/DXHook/CProxyDirect3D9.cpp (hunk 1 above)
Comment (G-Moris) [reply Client/core/DXHook/CProxyDirect3D9.cpp:185]: Is it required?

**Comment by T-MaxWiese-T** [comment]
So in the task manager sometimes the icon and also the subprocess name does not change when the additional subprocess `wow64_helper.exe` (helper library to run x86 applications on x64 versions of Microsoft Windows) starts automatically, which seems to happen randomly. However, the subprocess also closes again randomly, so that it works again as intended.

**Comment by G-Moris** [comment]
> So in the task manager sometimes the icon and also the subprocess name does not change when the additional subprocess `wow64_helper.exe` (helper library to run x86 applications on x64 versions of Microsoft Windows) starts automatically, which seems to happen randomly. However, the subprocess also closes again randomly, so that it works again as intended.

I didn't understand you

**Comment by T-MaxWiese-T** [comment]
> I didn't understand you

What exactly don't you understand?
I think the text I have written should be understandable.

**Pull request merged**